scripts/install-chef-suse.sh

```diff
@@ -344,18 +344,19 @@ EOF
     exit 1
 fi
 
-if ! (systemctl -q is-active crowbar-init.service && reset_crowbar ); then
+if ! (reset_crowbar 2> /dev/null); then
 
     cat <<EOF | pipe_show_and_log
 Aborting: Can not initialize Crowbar database
 
-Prior running the installation script, please execute at least
+Prior to running the installation script, please execute at least
+(c.f. Chapter 8 of the Deployment Guide):
 
   systemctl start crowbar-init.service
   crowbarctl database create
 
-If you configured an external database,
-there might be an error connecting.
+If you configured an external database, there might be an error
+connecting.
```

Comment:
After checking the intent of https://github.com/crowbar/crowbar/pull/2369 (which introduced this code and the bug), I would suggest this instead:

```bash
if ! reset_crowbar; then
    if systemctl -q is-active crowbar-init.service; then
        cat <<EOF | pipe_show_and_log
Aborting: Can not initialize Crowbar database

Prior to running the installation script, please execute at least
(c.f. Chapter 8 of the Deployment Guide):

  systemctl start crowbar-init.service
  crowbarctl database create

If you configured an external database, there might be an error
connecting.
EOF
    else
        cat <<EOF | pipe_show_and_log
Aborting: Can not initialize Crowbar database

If you configured an external database, there might be an error
connecting.
EOF
    fi
    exit 1
fi
```

The rationale is that we always want to do `crowbar_reset`; but the error message to provide clear guidance depends on the situation.

Reply:
Done.